.gitignore

```diff
@@ -1,5 +1,2 @@
 vul_data
 .DS_Store
-
```

Comment:
Why is anything more than just your yaml file being changed?

Reply:
Honestly I didn't make any of these changes. I don't know where they came from and they should probably be reverted, unless Meneely wants them.

Reply:
Yeah, I probably wouldn't include this in the commit. Should just be changes to the `.yml` I think

Reply:
I sent an email along to Rabb and Cioffi, hopefully they can give me some direction on what to do with these files.

Reply:
Per Rabb's comments in class today, I'm going to leave these alone.